content/stratisd-release-notes-3.6.5.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "stratisd 3.6.5 Release Notes"
+date = 2024-02-21
+weight = 37
+template = "page.html"
+render = true
++++
+
+*mulhern, Stratis Team*
+
+`stratisd` 3.6.5 includes a modification to its internal locking mechanism
+which allows a lock with a smaller scope to be taken when a global lock is
+blocked by relaxing the fairness restriction that would otherwise give that
+global lock precedence. It also includes a number of housekeeping commits and
+minor improvements.
```

Comment:
This is not entirely true. It doesn't just apply to global locks. For example, we would see the same behavior if we had a read lock on pool 1, then a queued write lock on pool 1, and even a read lock on pool 2. The basic idea is that lock acquisitions that conflict with the current lock are queued. The change allows locks that do not conflict with the currently acquired lock to skip the queue instead of being placed behind the queued conflicting locks.